cypress/integration/ete-okta/registration_1.2.cy.ts

```diff
@@ -219,7 +219,7 @@ describe('Registration flow - Split 1/2', () => {
 				const newReturnUrl = encodeURIComponent(
 					'https://www.theguardian.com/technology/2017/may/04/nier-automata-sci-fi-game-sleeper-hit-designer-yoko-taro',
 				);
-				cy.visit(`/welcome/${token}?returnUrl=${newReturnUrl}`);
+				cy.visit(`/welcome/${token}&returnUrl=${newReturnUrl}`);
```

Comment:
Wait, how does this work? Where does the query string start?

Reply:
so buckle up,

the regex matcher on like `216` is `/welcome\/([^"]*)/` which essentially captures everything after the `/welcome` and puts it into the "token" variable, but it's effectively the token + anything after the token, which includes query parameters!

and this change means that the `ref` and `refViewId` is being sent on most emails for tracking purposes, meaning that the query string starts in the "token" variable 😅 

so if you had a url like `https://profile.theguardian.com/welcome/some_token?foo=bar`

then the `token` variable on this line is set to `some_token?foo=bar` not just `some_token`, and the query string is started as part of the token variable